exercises/sum-of-multiples/src/example.h

```diff
@@ -0,0 +1,8 @@
+#ifndef _SUM_OF_MULTIPLES_H
+#define _SUM_OF_MULTIPLES_H
+
+#include <stddef.h>
+
+unsigned int sum_of_multiples_to(const unsigned int *multiples, const size_t number_of_multiples, const int up_to);
```

Comment:
I thought `sum_of_multples_to` would be more descriptive.  But now that I read it, I seems excessive.  I'll change it to `sum_of_multiples`